content/zh-cn/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -276,49 +378,54 @@ The following manifest defines an Ingress that sends traffic to your Service via
    Version: 1.0.0
    Hostname: web-55b8c6998d-8k564
    ```
+<!--
+1. Optionally, you can also visit `hello-world.example` from your browser.
+-->
+5. 可选步骤：你也可以从你的浏览器中访问 `hello-world.example`。
 
    <!--
-   You can also visit `hello-world.info` from your browser.
-
-   * **Optionally**
-
-     Look up the external IP address as reported by minikube:
-   -->
-
-   你也可以从浏览器访问 `hello-world.info`。
-
-   * **可选**
-
-     查看 Minikube 报告的外部 IP 地址：
-
-     ```shell
-     minikube ip
-     ```
-
-     <!--
-     Add line similar to the following one to the bottom of the `/etc/hosts` file on
+   Add a line to the bottom of the `/etc/hosts` file on
      your computer (you will need administrator access):
-     -->
-
-     将类似以下这一行添加到你计算机上的  `/etc/hosts` 文件的末尾（需要管理员访问权限）：
+   -->
+   在你的计算机上的 `/etc/hosts` 文件的最后添加一行（你需要管理员权限）：
 
-     ```none
-     172.17.0.15 hello-world.info
-     ```
+     {{< tabs name="hosts" >}}
+     {{% tab name="Linux" %}}
+   <!--
+   Look up the external IP address as reported by minikube
+   -->
+   查找 minikube 报告的外部 IP 地址：
+   ```shell
+   ```none
+     minikube ip 
```

Comment:
The preview is not good here: https://deploy-preview-46261--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/access-application-cluster/ingress-minikube/